examples/colab/object_detection.ipynb

```diff
@@ -80,20 +99,18 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 0,
       "metadata": {
-        "cellView": "form",
-        "colab": {},
+        "cellView": "both",
         "colab_type": "code",
-        "id": "6cPY9Ou4sWs_"
+        "id": "6cPY9Ou4sWs_",
+        "colab": {}
       },
-      "outputs": [],
       "source": [
         "#@title Imports and function definitions\n",
         "\n",
-        "# Currently %tensorflow_version 2.x installs beta1, which doesn't work here.\n",
-        "# %tensorflow_version can likely be used after 2.0rc0  \n",
-        "!pip install tf-nightly-gpu-2.0-preview\n",
+        "# Now stable version tensorflow 2.1.0 is there\n",
+        "\n",
+        "!pip install tensorflow==2.1.0\n",
```

Comment:
Does this still enable running on a GPU kernel? You can test by switching the kernels and measuring the time it takes. My guess is that tensorflow-gpu has to be installed.

Reply:
There is no separate installation of TensorFlow GPU in 2.1.0 version. Now _pip install tensorflow_ automatically installing both CPU and GPU version. And I just checked also, the kernel is running on GPU.

Reply:
Reference link - https://github.com/tensorflow/tensorflow/releases/tag/v2.1.0

Reply:
Sounds good, thanks for checking!